### ISSUE TITLE
How to pass to model compile multiple loss function?

### ISSUE THREAD

**Issue by Racines** [SciSharp/TensorFlow.NET#856]
Hello,
I want to pass multiple loss functions to my model but the compile method do not accept more than one loss function.

I want my two defined output use a different loss function
How can I achieve this?

from the [doc ](https://scisharp.github.io/Keras.NET/api/Keras.Models.BaseModel.html#Keras_Models_BaseModel_Compile_Keras_StringOrInstance_System_String_System_String___System_Single___System_String_System_String___Numpy_NDarray___) you can give a list or a dictionary of losses.

**Comment by CrimyTheBold** [comment]
I'm also interested to build a multi-head (value head + policy head) model
